docker/Dockerfile.cpu

```diff
@@ -2,7 +2,7 @@
 # This Dockerfile contains the Devito codes and can be built using different base images.
 ##############################################################
 
-ARG pyversion=python:3.7
+ARG pyversion=python:3.8
```

Comment:
Why not get ahead of the curve and use Python 3.9 or 3.10 here (and elsewhere)? Looks like the GPU one uses 3.9

Reply:
I am open to this, let's see what others think

Reply:
3.9 would be a safe option yes